filter.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const { config } = require('./config')
+
+const minimatch = require('minimatch')
+const path = require('path')
+
+/**
+ * Checks, whenever a file should be excluded by the,
+ * excludes provided by the user
+ * @param {String} filePath path to the file which will be checked
+ * @param {String[]} excludes array of exclude rules against whom
+ * the file will be checked
+ */
```

Comment:
I'd suggest we tighten the language up a bit here:
```JavaScript
/**
 * Checks whether a file should be excluded from processing
 * @param {String} filePath path of file to check
 * @param {String[]} excludes array of exclusion rules to compare to
 */
```

Reply:
A good suggestion. It's hard for me to write short descriptions for the parameters and return values.
I will try better. :))